lib/client.js

```diff
@@ -25,134 +35,135 @@ exports.create = function (config) {
           'Content-Length': (data) ? Buffer.byteLength(data, 'utf8') : 0,
           'User-Agent': 'recurly-js/' + pjson.version
         }
-      };
+      }
 
       if (headers) {
-        for (var p in headers) {
-          if (!headers.hasOwnProperty(p)) continue;
+        for (const p in headers) {
```

Comment:
I think since we're not reassigning it in the block, it should be `const` per: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of